packages/components/src/color-picker/component.tsx

```diff
@@ -57,23 +59,31 @@ const ColorPicker = (
 ) => {
 	const {
 		enableAlpha = false,
-		color,
+		color: colorProp,
 		onChange,
 		defaultValue = '#fff',
 		copyFormat,
 		...divProps
 	} = useContextSystem( props, 'ColorPicker' );
 
+	const [ color, setColor ] = useControlledValue( {
+		onChange,
+		value: colorProp,
+		defaultValue,
+	} );
+
 	// Use a safe default value for the color and remove the possibility of `undefined`.
 	const safeColordColor = useMemo( () => {
 		return color ? colord( color ) : colord( defaultValue );
```

Comment:
Given that useControlledValue, is already taking into account defaultValue, I guess we can simplify this to `return colord( color )`.

Reply:
Good suggestion, thanks! I've simplified the return statement here and moved the inline comment to above `useControlledValue` as that's where the default value logic is now happening.